lighthouse-extension/app/src/popup.js

```diff
@@ -51,7 +51,8 @@ document.addEventListener('DOMContentLoaded', _ => {
 
     let qsBody = '**Lighthouse Version**: ' + getLightHouseVersion() + '\n';
     qsBody += '**Chrome Version**: ' + getChromeVersion() + '\n';
-    qsBody += '**Stack Trace**:\n' + err.stack;
+    qsBody += '**Error Message**: ' + err.message + '\n';
+    qsBody += '**Stack Trace**:\n ```' + err.stack + '```';
```

Comment:
yeah, it could be invasive. Most of the extension issues we've seen with an actual caught error (as opposed to something more subtle, like -1 in FMP) have been tracked down without the URL. We can add the optional prompt for it later if we find ourselves asking for it more often than not